Website: Make all documentation navigation on fleetdm.com sticky

eashaw opened the pull request:
Closes #5650 

Changes:
- updated the docs navigation to use `position: sticky`
- Updated the script in `layout.ejs` to scroll the top docs navigation when the header is hidden

mike-j-thomas commented:
Thanks, @eashaw. There's no need for the breadcrumbs and search bar to be sticky. Only the left and right side navigation.

I sent an invite to meet tomorrow to go over the interactions and make sure that the scrolls work as expected - particularly on very short browser heights because it looks like the bottoms are getting chopped.

mikermcneil reviewed (changes requested):
Lint issue, fyi

<img width="623" alt="image" src="https://user-images.githubusercontent.com/618009/167943598-46eb6e49-1c9f-45c1-96fe-c85a38fd5437.png">

eashaw commented:
@mike-j-thomas I updated this PR with the following changes:
- Removed `position: sticky` from the top navigation
- Added padding when scrolling to an anchor link

mike-j-thomas commented:
This is looking better @eashaw. The scroll on small browser windows does not display the last items of the navigation. https://www.loom.com/share/6cf2772c735c463498ace147e6174a7f
